lib/phoenix_live_view/engine.ex

```diff
@@ -322,7 +322,8 @@ defmodule Phoenix.LiveView.Engine do
   def handle_end(state, opts \\ []) do
     %{static: static, dynamic: dynamic} = state
     safe = {:safe, Enum.reverse(static)}
-    meta = [live_rendered: true] ++ Keyword.get(opts, :meta, [])
+    meta = [live_rendered: true, no_dynamics: dynamic == []] ++ Keyword.get(opts, :meta, [])
```

Comment:
In theory we don't need to pass `no_dynamics`. We could simply check for the block being a single element list?

Reply:
Ignore me... there is more at play... let me think about it.